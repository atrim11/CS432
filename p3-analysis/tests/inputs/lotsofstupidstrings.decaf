def int main() {
    processString("Hello, World!");
    concatStrings("foo", "bar");
    formatString("Value: %d", 42);
    checkSubstring("Hello", "lo");
    toUpperCase("lowercase");
    findCharacter("example", "e");
    splitString("a,b,c", ",");
    replaceSubstring("Hello, World!", "World", "C++");
    trimWhitespace("   whitespace   ");
    joinStrings("one", "two", "three");
    getStringLength("length test");
    substring("substring example", 0, 9);
    compareStrings("abc", "abc");
    stringContains("test", "es");
    stringStartsWith("hello", "he");
    stringEndsWith("world", "ld");
    return 0;
}