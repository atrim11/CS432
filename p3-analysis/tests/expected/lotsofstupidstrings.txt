Symbol 'processString' undefined on line 2
Symbol 'concatStrings' undefined on line 3
Symbol 'formatString' undefined on line 4
Symbol 'checkSubstring' undefined on line 5
Symbol 'toUpperCase' undefined on line 6
Symbol 'findCharacter' undefined on line 7
Symbol 'splitString' undefined on line 8
Symbol 'replaceSubstring' undefined on line 9
Symbol 'trimWhitespace' undefined on line 10
Symbol 'joinStrings' undefined on line 11
Symbol 'getStringLength' undefined on line 12
Symbol 'substring' undefined on line 13
Symbol 'compareStrings' undefined on line 14
Symbol 'stringContains' undefined on line 15
Symbol 'stringStartsWith' undefined on line 16
Symbol 'stringEndsWith' undefined on line 17
